bigchaindb/log/configs.py

```diff
@@ -18,9 +18,8 @@
     'formatters': {
         'console': {
             'class': 'logging.Formatter',
-            'format': (
-                '%(name)-15s %(levelname)-8s %(processName)-10s %(message)s'
-            ),
+            'format': ('[%(asctime)s] [%(levelname)s] (%(name)s) '
+                       '%(message)s (%(processName)-10s - pid: %(process)d)'),
```

Comment:
Is `asctime` UTC or local time?
It will matter when aggregating logs for alerts.